javascript/ql/lib/semmle/javascript/security/dataflow/TaintedPathCustomizations.qll

```diff
@@ -892,7 +892,10 @@ module TaintedPath {
       TaintTracking::uriStep(node1, node2)
       or
       exists(DataFlow::CallNode decode |
-        decode.getCalleeName() = "decodeURIComponent" or decode.getCalleeName() = "decodeURI"
+        decode.getCalleeName() = "decodeURIComponent" or
+        decode.getCalleeName() = "decodeURI" or
+        decode.getCalleeName() = "escape" or
+        decode.getCalleeName() = "unescape"
```

Comment:
```suggestion
        decode = DataFlow::globalVarRef([
            "decodeURIComponent",
            "decodeURI",
            "escape",
            "unescape"])
          .getACall()
```
Sorry I just noticed that this code didn't actually check what the receiver of the call is. This was unlikely to be a problem for the names used so far, but I could see it being a problem that we now match any function called `escape`.